### ISSUE TITLE
Cyclic relationship in RXNO_0000580

### ISSUE THREAD

**Issue by rebecca-foulger** [rsc-ontologies/rxno#49]
Hi,

There looks to be a cyclic relationship in RXNO_0000580, where RXNO_0000580 is a subClass of itself:

```
<!-- http://purl.obolibrary.org/obo/RXNO_0000580 -->

    <owl:Class rdf:about="http://purl.obolibrary.org/obo/RXNO_0000580">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/RXNO_0000140"/>
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/RXNO_0000580"/>
```
Please could you take a look?

Many thanks,
Rebecca

**Comment by StroemPhi** [comment]
@rebecca-foulger thanls alot for finding this!
@colinbatchelor fixed in #50

**Comment by rebecca-foulger** [comment]
Thanks for fixing this so quickly!
